src/com/sbezboro/standardplugin/listeners/DeathListener.java

```diff
@@ -148,13 +153,22 @@ public void onPlayerDeath(PlayerDeathEvent event) {
 			victim.setLastDeathInPvp(true);
 			victim.setLastDeathBySpawnkill(false);
 		} else {
+			if (victim.wasInPvp()) {
+				indirect_killer = plugin.getStandardPlayerByUUID(victim.getLastAttackerUuid());
+				deathMessage += ", while in PVP with " + indirect_killer.getDisplayName();
+			}
 			victim.setLastDeathInPvp(victim.wasInPvp());
 			victim.setLastDeathBySpawnkill(false);
 		}
-
 		DeathEvent deathEvent = new DeathEvent(victim);
 		deathEvent.log();
 
+		if (indirect_killer != null) {
+			// code copied from private DeathEvent::log
+			HttpRequestManager.getInstance().startRequest(
+					new DeathHttpRequest(victim.getUuidString(), "player", indirect_killer.getUuidString()));
```

Comment:
This will cause two death events for one death in this case. I'm going to tackle this and rework how `DeathEvent` works to make this a bit nicer

Reply:
yes, definitely not the cleanest solution (just the quickest :-)